SCons/Script/Main.xml

```diff
@@ -824,6 +824,18 @@ which corresponds to <option>--silent</option>.
 </listitem>
 </varlistentry>
 <varlistentry>
+<term><literal>no_progress</literal></term>
+<listitem>
+<para>
+which corresponds to -Q.
+</para>
+<para>
+    Note: The initial progress output will still be output as this is done before the SConstruct/SConscript which contains the SetOption is processed
+    <literal>scons: Reading SConscript files ...</literal>
+</para>
+</listitem>
```

Comment:
With grudging acuiqesance from @bdbaddog I've been including the following when I think of it - he doesn't seem convinced this belongs in the manpage but hasn't rejected a couple of PRs which include it; I think it's helpful.
```
<para><emphasis>Available since &scons; 4.0.</emphasis></para>
```